game/songparser-ini.cc

```diff
@@ -1,7 +1,6 @@
 #include "songparser.hh"
 
 #include <boost/algorithm/string.hpp>
-#include <boost/regex.hpp>
```

Comment:
whut? these files didn't need it in the first place?

Reply:
correct.